xbmc/cores/RetroPlayer/streams/RetroPlayerAudio.h

```diff
@@ -60,8 +60,8 @@ namespace RETRO
 
   private:
     CRPProcessInfo& m_processInfo;
-    IAEStream* m_pAudioStream;
-    bool m_bAudioEnabled;
+    IAEStream*      m_pAudioStream;
+    bool            m_bAudioEnabled;
```

Comment:
Unrelated cosmetics. Also our coding style is to avoid mid-line alignment. Much of my code departs from this because it was written so long ago (header dates are accurate)